docs/source/cdk/index.rst

```diff
@@ -0,0 +1,5 @@
+.. toctree::
```

Comment:
I'd suggest adding a title here just so it's clear what this documentation is. Something like 

```
CDK Deployment 
=============
```